tsconfig.json

```diff
@@ -13,7 +13,8 @@
         "declaration": true,
         "noImplicitAny": false,
         "noEmitOnError": false,
-        "sourceMap": true
+        "sourceMap": true,
+        "skipLibCheck": true,
```

Comment:
why do we disable lib check?
i understand it may be useful when doing the bump (faster ts build), but generalyl we should keep it (and "resolve" dependency if multiple copies of the same lib are present) ?